configs/body_2d_keypoint/integral_regression/coco/ipr_res50_8xb64-210e_coco-256x256.py

```diff
@@ -17,7 +17,7 @@
     dict(
         type='MultiStepLR',
         begin=0,
-        end=210,
```

Comment:
What about using the default value `infinite`?

Reply:
I think `end=train_cfg['max_epochs']` is more intuitive for users to understand